### ISSUE TITLE
HEADER_SIZE = 80 is stringent at my case

### ISSUE THREAD

**Issue by la7low** [WoLpH/numpy-stl#32]
Hello Mr Wolph,

The default 80 bytes, i.e. 80 characters were quite stringent for me in my use case as I had some stl parts with more characters. Would you consider increasing it to 256? I think it would not affect the speed adversely too much.

Cheers
Lazlo

**Comment by WoLpH** [comment]
Hi Lazlo,

Do you mean the 80 character header limit for binary files? That's not a performance decision but it's how the standard appears to be defined: https://en.wikipedia.org/wiki/STL_(file_format)#Binary_STL
With the binary format it has to be a fixed amount of characters since the file will be read incorrectly otherwise.

Cheers

**Comment by WoLpH** [comment]
Monkeypatching it is fairly easy btw:

```
from stl import stl
stl.HEADER_SIZE = 256
```

But your files will be incompatible with most (all?) STL libraries

**Comment by la7low** [comment]
Ahh ok, I did not know that has to be fixed at binary stls.
But what I wrote still applies to ASCII stls (my use case) and the end of the part name might be stripped off (if the name is longer than 80 - len('solid ') characters).

I need to think about it more after work but i would still suggest some modification at the beginning of the _ascii_reader method that it recovers part names which are more than 74 chars long.
Obviously I monkeypatched it for my use case just as you wrote but others might also have long part names but not willing take some effort to check what is going on.
